guides/source/documents.yaml

```diff
@@ -140,6 +140,11 @@
       work_in_progress: true
       url: api_app.html
       description: This guide explains how to effectively use Rails to develop a JSON API application.
+    -
+      name: Active Record and PostgreSQL
+      work_in_progress: true
+      url: active_record_postgresql.html
+      description: This guide covers PostgreSQL specific usage of Active Record.
```

Comment:
I agree with @rafaelfranca. This document is nowhere near the point where we can publish it as a WIP. Sadly I don't have time at the moment to continue working on this guide.